src/strategies/F.cpp

```diff
@@ -0,0 +1,81 @@
+#include "F.h"
+#include "objects/ParticleContainer.h"
+#include "utils/ArrayUtils.h"
+#include <functional>
+
+void calculateF_Verlet(ParticleContainer &particles, double, double) {
+    for (auto &p1 : particles) {
+        for (auto &p2 : particles) {
+            if (p1 != p2) {
+                p1.setF(p1.getF() + ArrayUtils::elementWiseScalarOp(
+                                        p1.getM() * p2.getM() / std::pow(ArrayUtils::L2Norm(p1.getX() - p2.getX()), 3),
+                                        p2.getX() - p1.getX(), std::multiplies<>()));
+            }
+        }
+    }
+}
+
+void calculateF_VerletThirdLaw(ParticleContainer &particles, double, double) {
+    // loop over unique pairs
+    for (size_t i = 0; i < particles.size(); ++i) {
+        auto &p1 = particles[i];
+
+        // avoid recalculating force for pairs which have already been computed
+        for (size_t j = i + 1; j < particles.size(); ++j) {
+            auto &p2 = particles[j];
```

Comment:
About this, I would say, you can leave it the way it is. Especially because you would need changes with respect to LinkedCells anyways. So, focus on that :)